controllers/operator/routes.go

```diff
@@ -555,6 +566,18 @@ func (r *AuthenticationReconciler) newRoute(authCR *operatorv1alpha1.Authenticat
 			InsecureEdgeTerminationPolicy: routev1.InsecureEdgeTerminationPolicyRedirect,
 			DestinationCACertificate:      string(fields.DestinationCAcert),
 		}
+	} else if fields.RoutePath == "" {
+		// Passthrough route (if RoutePath is empty)
+		route.Spec.TLS = &routev1.TLSConfig{
+			Termination:                   routev1.TLSTerminationPassthrough,
+			InsecureEdgeTerminationPolicy: routev1.InsecureEdgeTerminationPolicyRedirect,
+		}
+	} else {
```

Comment:
As discussed we can remove it for now as it is not used.

Reply:
@ujjwalchk-it , removed the edge termination block